ppr-ui/src/components/unitNotes/UnitNoteReview.vue

```diff
@@ -198,7 +198,7 @@ export default defineComponent({
 
     const localState = reactive({
       validateSubmittingParty: false,
-      initialAttention: getMhrUnitNoteRegistration.value?.attentionReference,
+      initialAttention: isCancelUnitNote ? '' : getMhrUnitNoteRegistration.value?.attentionReference,
```

Comment:
As Dima mentioned in his review, you'll need to a property accessor for `.value`
(ie dot notation is pretty standard for us)

The reason why you wouldn't see an error for this, `isCancelUnitNote` is essentially just a permanent `true` boolean in the the context of this ternary(it's defined in the composable, so always true, not null or undefined etc).
Due to this, you wouldn't see a type error or anything in the console, it just always resolves to the first condition, ie ''. 

You will notice this in the UI however, where it seems to rely on the getter when navigating to and from the Review Confirm screen of a base unit note, to preserve the captured value (before submission). 

Give it a look and you will see what he means!

Reply:
If your interested in more context on the **why** we need to do this, see vue's explanation of computed refs: https://vuejs.org/guide/essentials/computed.html

Reply:
Thanks for elaborating @cameron-eyds

Reply:
Great!
I will fix them